src/types.ts

```diff
@@ -101,7 +101,7 @@ declare namespace Bittrex {
     QuantityRemaining: Decimal; // 100000.00000000
     Commission: Decimal; // 0.00000000
     Price: Decimal; // 0.00000000
-    PricePerUnit: null; // null
+    PricePerUnit: Decimal; // 0.00000000
```

Comment:
I think this is actually `Decimal | null`. I think there's more fields here that need to be updated.

Reply:
Why would it be null? This is the OrderHistory, there better be a PricePerUnit. LOL.

Reply:
It's null when the order isn't filled yet. I just had this happen to me in a test case. Basically when QuantityRemaining > 0.

Reply:
Fuck. You'd think `history` would be orders that are completed. Ugh. Amateur hour on the API design. 

GET for all the methods, including orders! =(